### DISCUSSION TITLE
ui上无法看到节点列表

### DISCUSSION THREAD

**Discussion by HongKyun520** [weibocom/rill-flow#70] [Q&A] [answered]
我按照文档上的流程在本机使用docker启动了rill-Flow的ui界面，但是在编辑流程的时候遇到了一些问题，具体为"节点列表"中的可选节点为空，不知道这是什么原因
![Uploading iShot_2024-07-05_15.55.30.png…]()

**Comment by HongKyun520** [comment]
补充图片
![iShot_2024-07-05_15 55 30](https://github.com/weibocom/rill-flow/assets/66047803/89960abc-12a5-402b-8d18-6bbd619720be)

**Comment by qdaxb** [comment]
ping @qiaofenlin

**Comment by qiaofenlin** [comment, marked as answer]
@HongKyun520 进入该页面是否有报错？ 这个列表没有数据可能是项目连接mysql有异常。 可以在`docker_rill-flow_1`容器中查看`/usr/local/tomcat/logs/info.log`的具体异常日志。
